csharp/Ice/Greeter/Server/Chatbot.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+using VisitorCenter;
+
+namespace GreeterServer;
+
+/// <summary>A Chatbot is an Ice servant that implements Slice interface Greeter.</summary>
+internal class Chatbot : GreeterDisp_
+{
+    // Implements the abstract method greet from the GreeterDisp_ class generated by the Slice compiler.
+    public override string greet(string name, Ice.Current current)
```

Comment:
This is a "sync" dispatch, which is the simplest, and certainly ok when the dispatch is indeed sync.

Should I add a GreeterAsync demo that shows AMD, similar to the C++ greeterAsync demo?

Reply:
I wouldn't.